backend/node/genvm/base.py

```diff
@@ -170,33 +176,47 @@ async def deploy_contract(
                 )
 
             except Exception as e:
-                print("Error deploying contract", e)
                 trace = traceback.format_exc()
-                print(trace)
                 error = e
```

Comment:
```suggestion
            except Exception as error:
                trace = traceback.format_exc()
```

Reply:
We want this to be in the developer terminal, why are you suggesting to remove it?

Reply:
actually need the remapping of `error`  here because it is used down below when returning the receipt